pkgdb2client/cli.py

```diff
@@ -781,11 +778,8 @@ def main():
         LOG.setLevel(logging.INFO)
 
     if arg.test:
-        global fasclient, pkgdbclient
+        global pkgdbclient
         print "Testing environment"
-        fasclient = AccountSystem(
```

Comment:
``--insecure``, ``--fasurl`` and ``--pkgdburl`` seem like a good idea but would you mind if I put it in a separate pull-request as this will impact ``cli.py`` as well?